### PR TITLE
Fix json parser end line

### PR DESCRIPTION
**Pull request by achiar99**
**By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

## Description
In some cases the end_mark line of json object was wrong.
This should fix it

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my feature, policy, or fix is effective and works
- [x] New and existing tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: tests/common/parsers/json/test_json_parser.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+class TestJsonParser(unittest.TestCase):
+    def test_json_parser_lines(self):
+        test_file = current_dir + "/resources/file1.json"
+        cfn_dict, cfn_str = parse(test_file)
+
+        resources = cfn_dict.get('Resources')
+        bucket = resources.get('S3Bucket')
+        properties = bucket.get('Properties')
+
+        assert resources.start_mark.line == 5
```
Comment (tronxd) [inline tests/common/parsers/json/test_json_parser.py:17]: @achiar99 Isn't the resources section start in line 2?

File: tests/common/parsers/json/test_json_parser.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        properties = bucket.get('Properties')
+
+        assert resources.start_mark.line == 2
+        assert resources.end_mark.line == 31
+
+        assert bucket.start_mark.line == 3
+        assert bucket.end_mark.line == 30
+
+        assert properties.start_mark.line == 5
+        assert properties.end_mark.line == 28
```
Comment (tronxd) [inline tests/common/parsers/json/test_json_parser.py:24]: @achiar99 Looks like that all end line numbers should be greater by 1 according to the test file

Comment (achiar99) [reply tests/common/parsers/json/test_json_parser.py:24]: yes.. before my fix it was 27, 29, 30.. and all the remediations was wrong.
this way everything looks ok.

**Review by gruebel — APPROVED** [review]
nice and easy 🍺

**Pull request closed without merging**